### ISSUE TITLE
model paddle has no attribute 'incubate'

### ISSUE THREAD

**Issue by JITESH11989** [PaddlePaddle/PaddleOCR#2751]
Global:
  use_gpu: true
  epoch_num: 72
  log_smooth_window: 20
  print_batch_step: 5
  save_model_dir: ./output/rec_devanagari_lite_SRN
  save_epoch_step: 3
  eval_batch_step:
  - 0
  - 5000
  cal_metric_during_train: true
  pretrained_model: null
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: null
  character_dict_path: ppocr/utils/dict/devanagari_dict.txt
  character_type: devanagari
  max_text_length: 50
  num_heads: 8
  infer_mode: false
  use_space_char: true
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  clip_norm: 10.0
  lr:
    name: Cosine
    learning_rate: 0.001
  regularizer:
    name: L2
    factor: 1.0e-05
Architecture:
  model_type: rec
  algorithm: SRN
  in_channels: 1
  Transform: null
  Backbone:
    name: ResNetFPN
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 48
  Head:
    name: SRNHead
    max_text_length: 50
    num_heads: 8
    num_encoder_TUs: 2
    num_decoder_TUs: 4
    hidden_dims: 512
    fc_decay: 1.0e-05
Loss:
  name: SRNLoss
PostProcess:
  name: SRNLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
Train:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/
    label_file_list:
    - train_data/train_rec_gt.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecAug: null
    - SRNLabelEncode: null
    - SRNRecResizeImg:
        image_shape:
        - 3
        - 32
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
        - encoder_word_pos
        - gsrm_word_pos
        - gsrm_slf_attn_bias1
        - gsrm_slf_attn_bias2
  loader:
    shuffle: False
    batch_size_per_card: 32
    drop_last: False
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/
    label_file_list:
    - train_data/test_rec_gt.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - SRNLabelEncode: null
    - SRNRecResizeImg:
        image_shape:
        - 3
        - 32
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
        - encoder_word_pos
        - gsrm_word_pos
        - gsrm_slf_attn_bias1
        - gsrm_slf_attn_bias2
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 32
    num_workers: 4

**Comment by LDOUBLEV** [comment]
Please update your paddlepaddle. Then the problem will be solved.
